src/printer.js

```diff
@@ -138,40 +152,43 @@ const print = (path, opts, print) => {
         return group(path.map(print, 'children'));
       }
     }
-    case 'Text':
-      if (!isPreTagContent(path)) {
```

Comment:
This logic isn’t changed at all; I just flattened the `if` statements to be a little more readable as I was debugging this behavior.